manifests/m/moneymanagerex/moneymanagerex/1.6.4/moneymanagerex.moneymanagerex.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: moneymanagerex.moneymanagerex
+PackageVersion: 1.6.4
+PackageLocale: en-US
+Publisher: Money Manager EX

```

```suggestion
Publisher: Humanity
```
This must match control panel